arch/tools/build_cli.sh

```diff
@@ -1,23 +1,4 @@
 #!/bin/bash
 
-# Define paths
-source_schema="../arch_config_schema.yaml"
```

Comment:
Its' unclear to me how the config_generator.py gets access to this schema to validate the arch_config from the developer. We have to have the cli run independently - meaning it should not require a fork of main. For those who want to `archgw build` from source, its necessary for them to fork main and run locally.

Reply:
schema file is burned into docker image - see here https://github.com/katanemo/archgw/blob/main/arch/Dockerfile#L26